recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 
 requirements:
   host:
-    - jupyter-packaging >=0.7.9,<1.0
+    - jupyter-packaging 0.7.9
     - jupyterlab >=3.0.0,<4.0.0
```

Comment:
isn't that we want to pin to specific version lowest common for all architectures ?